projector.py

```diff
@@ -71,8 +73,11 @@ def set_network(self, Gs, minibatch_size=1):
         # Find dlatent stats.
         self._info('Finding W midpoint and stddev using %d samples...' % self.dlatent_avg_samples)
         latent_samples = np.random.RandomState(123).randn(self.dlatent_avg_samples, *self._Gs.input_shapes[0][1:])
-        dlatent_samples = self._Gs.components.mapping.run(latent_samples, None) # [N, 18, 512]
-        self._dlatent_avg = np.mean(dlatent_samples, axis=0, keepdims=True) # [1, 18, 512]
+        if self.tiled:
+            dlatent_samples = self._Gs.components.mapping.run(latent_samples, None)[:, :1, :] # [N, 1, 512]
+        else:
+            dlatent_samples = self._Gs.components.mapping.run(latent_samples, None) # [N, 18, 512]
+        self._dlatent_avg = np.mean(dlatent_samples, axis=0, keepdims=True) # [1, 1 or 18, 512]
```

Comment:
Would you mind explaining the idea behind tiled mode?

Compared to the original repository by Nvidia, I have noticed that @rolux changed this line.
Then you offered to add an argument to toggle between the original line and the line by @rolux.

I suspect tiled mode leads to better projection results, but I don't know why.